ReactSkia/pluginfactory/RnsPluginFactory.cpp

```diff
@@ -41,12 +59,26 @@ std::unique_ptr<RNSApplicationManagerInterface> RnsPluginFactory::createAppManag
 }
 #endif // RNSP_APP_MANAGER
 
+#if ENABLE(RNSP_PLATFORM_MANAGER)
+std::unique_ptr<RNSPlatformManagerInterface> RnsPluginFactory::createPlatformManagerHandle(PlatformCallbackClient &client) const {
+  if(platformManagerCreator_ == nullptr)
+    return nullptr;
```

Comment:
Coding standard change suggestion : enclose return into {}. Same for "createAppManagerHandle"